### PR TITLE
fix travis (flake8)

### PR DESCRIPTION
**Pull request by gurneyalex**

### CODE REVIEW

**Comment by tafaRU** [comment]
:+1:

File: sale_order_partial_invoice/sale.py [hunk 1]
```diff
@@ -47,12 +47,14 @@
 
 """
 import logging
-_logger = logging.getLogger(__name__)
```
Comment (yvaucher) [inline sale_order_partial_invoice/sale.py:50]: Don't we want to keep a blank line here to separate openerp import from lib imports?

Comment (gurneyalex) [inline sale_order_partial_invoice/sale.py:50]: @yvaucher done

**Comment by yvaucher** [comment]
:+1:

**Comment by pedrobaeza** [comment]
:+1:

**Pull request merged**